cli/add/add.go

```diff
@@ -106,6 +106,10 @@ func HandleAdd(args []string) (int, error) {
 
 // TODO(siggisim): Support specifying a version.
 func FetchModuleOrDisambiguate(module string) (string, *RegistryResponse, error) {
+	module = strings.Replace(module, "https://", "", 1)
+	module = strings.Replace(module, "github.com/", "github/", 1)
```

Comment:
For the modules on registry.build (which we're fetching here) the urls are `github/foo/bar` e.g. https://registry.build/github/bazelbuild/rules_go

Reply:
Since the registry supports the ".json" trick for API URLs, I feel like it would be slightly more ergonomic to avoid having a mapping from short names to domain names - that way people can just plug in a repo URL to the API, instead of having to strip ".com"

Reply:
https://registry.build/github.com/bazelbuild/rules_go/.json is a valid URL, fwiw

Reply:
Are you suggesting the urls should look like this? https://registry.build/github.com/bazelbuild/rules_go

Reply:
Yeah - fwiw pkg.go.dev does that `https://pkg.go.dev/github.com/drone/go-github/github`

Reply:
I just think those nested URLs are kinda ugly and I'd like to in the future potentially have other prefixes that are not urls.

I'd rather get rid of the github bit altogether and just have it be:
```
registry.build/bazelbuild/rules_go
```

But I added the `/github/` piece in case we want to add other things in the future (for example there's a bcr modules namespace that lives under `/bazel/` i.e. https://registry.build/bazel/aspect_rules_js/) 

The registry and the api are all static files at the moment and generated at build time, so I'd rather just have one canonical format for now. I can make those domain URLs valid in addition to the short name urls in the future if it becomes useful.